scripts/test.sh

```diff
@@ -27,12 +27,14 @@ if [ "$HELP" = true ]; then
 fi
 
 # Directory above this script
-RELAYER_PATH=$(
+root=$(
     cd "$(dirname "${BASH_SOURCE[0]}")"
     cd .. && pwd
 )
-source "$RELAYER_PATH"/scripts/constants.sh
+source "$root"/scripts/constants.sh
 
 go build -o tests/cmd/decider/decider ./tests/cmd/decider/
 
+"$root"/scripts/generate.sh
```

Comment:
I would still rather call this manually than have it be re-generated automatically ever. If it's quick it might not make too much of a difference. not a blocker though

Reply:
I agree. I wouldn't ever expect running a test to alter my git status